#
# Submitted-By: Marc Kleine-Budde, 2006-02-28
# Committed-By: Marc Kleine-Budde
#
# Error:
#
# freetype-config is not $SYSROOT aware
#
# Description: 
#
# this patch adds $SYSROOT support to freetype-config
#
# State:
# 
# not commited, but should be :)
#
Index: freetype-2.1.9/builds/unix/freetype-config.in
===================================================================
--- freetype-2.1.9.orig/builds/unix/freetype-config.in
+++ freetype-2.1.9/builds/unix/freetype-config.in
@@ -1,6 +1,6 @@
 #! /bin/sh
 
-prefix=@prefix@
+prefix=${SYSROOT}@prefix@
 exec_prefix=@exec_prefix@
 exec_prefix_set=no
 includedir=@includedir@
@@ -118,7 +118,7 @@ if test "$echo_libs" = "yes" ; then
   if test "$enable_shared" = "yes" ; then
     eval "rpath=\"$hardcode_libdir_flag_spec\""
   fi
-  libs="-lfreetype @LIBZ@"
+  libs="-Wl,-lfreetype @LIBZ@"
   if test "$libdir" != "/usr/lib" ; then
     echo -L$libdir $rpath $libs
   else
